src/index.ts

```diff
@@ -7,30 +7,31 @@ import { syncAugurNodeWithBlockchain } from "./blockchain/sync-augur-node-with-b
 import { runWebsocketServer } from "./server/run-websocket-server";
 import { ErrorCallback } from "./types";
 
+// tslint:disable-next-line:no-var-requires
 const { augurDbPath, ethereumNodeEndpoints, uploadBlockNumbers, websocketPort } = require("../config");
 
-var db: Knex;
-if (process.env["DATABASE"] == null) {
+let db: Knex;
+if (process.env.DATABASE) {
+  db = Knex({
+    client: "pg",
+    connection: process.env.DATABASE
+  });
+} else {
   sqlite3.verbose();
   db = Knex({
     client: "sqlite3",
     connection: {
       filename: augurDbPath
     },
-    acquireConnectionTimeout: 5*60*1000
-  });
-} else {
-  db = Knex({
-    client: "pg",
-    connection: process.env["DATABASE"]
+    acquireConnectionTimeout: 5 * 60 * 1000
   });
 }
 
 const augur: Augur = new Augur();
 
 augur.rpc.setDebugOptions({ broadcast: false });
 
-checkAugurDbSetup(db, ErrorCallback  => {
+checkAugurDbSetup(db, (err?: Error|null)  => {
```

Comment:
Hm, `ErrorCallback` is the same as `(err?: Error|null)` (except with `: void` return value) -- do we not want the void return here for some reason?

Reply:
Also, there's an extra space before the arrow, dunno if there's a jslint rule for that.